### PR TITLE
feat: update dlstreamer to use 2024.1 image

### PR DESCRIPTION
**Pull request by seanohair22**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

- Updates dlstreamer image to use 2024.1.1-dev
- Cleans up a warning about casing of `as` in Dockerfile

## Test Instructions if applicable

- Run `make run-demo`

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
https://github.com/intel-retail/performance-tools/pull/58

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]

File: helm/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM intel/dlstreamer:2023.0.0-ubuntu22-gpu682-dpcpp as base
+FROM intel/dlstreamer:2024.1.1-dev-ubuntu22 AS base
```
Comment (jim-wang-intel) [inline helm/Dockerfile:7]: do they have tag related to gpu dpcpp ? or they are all fold into one with dev-ubuntu22 now?

Comment (seanohair22) [reply helm/Dockerfile:7]: I looked and did not find a tag with gpu-dpcpp... I am attempting to test with Arc GPU to verify that it works

Comment (jim-wang-intel) [reply helm/Dockerfile:7]: chat with Brian, and currently this image seems to support our use cases. but it never hurts to test GPU I guess.

**Review by jim-wang-intel — APPROVED** [review]

File: helm/Dockerfile (hunk 1 above)
Comment (seanohair22) [reply helm/Dockerfile:7]: Yes, yolov5 works with CPU (scope is only yolov5 for now). I added a backlog item for GPU support

**Pull request merged**